src/dcqc/tests/libtiff_info_test.py

```diff
@@ -4,13 +4,16 @@
 
 class LibTiffInfoTest(ExternalBaseTest):
     tier = 2
+    pass_code = "0"
     target: SingleTarget
 
     def generate_process(self) -> Process:
         path = self.target.file.stage()
+        string_path = self.short_string_path(path, "dcqc-staged-")
```

Comment:
Right now, I think "dcqc-staged" is hardcoded in the package, I know there's a TODO to include this as a constant - I'll file a Jira ticket for it.